certified-connectors/DocuSign/apiDefinition.swagger.json

```diff
@@ -889,6 +890,221 @@
+            "name": "recordType",
+            "in": "query",
+            "required": true,
+            "enum": [
+              "account",
+              "opportunity",
+              "lead"
+            ],
+            "type": "string"
+          },
+          {
+            "name": "recordId",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "startDateTime",
+            "description": "Enter start date-time in the format: 2017-05-02T01:44Z",
+            "in": "query",
+            "type": "string",
+            "required": false,
+            "format": "date-time"
+          },
+          {
+            "name": "endDateTime",
+            "description": "Enter end date-time in the format: 2017-05-02T01:44Z",
+            "in": "query",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "top",
+            "description": "Top envelopes to be returned",
+            "in": "query",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "skip",
+            "description": "Envelopes to be skipped",
+            "in": "query",
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "crmType",
+            "in": "query",
+            "enum": [
+              "Dynamics365"
+            ],
+            "type": "string"
+          },
+          {
+            "name": "crmOrgUrl",
+            "in": "query",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ActivityListResponseEnvelope"
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+                "$ref": "#/definitions/ApiError"
+            }
+          },
+          "500": {
+            "description": "Server Error",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+              }
+            }
+          },
+        "x-ms-openai-data": {
+        "openai-enabled": true
+        }
+      }
+    },
+    "/accounts/salesCopilotAccount/envelopes/getRelatedRecords": {
+      "get": {
+        "tags": [
+          "DocuSign"
+        ],
+        "summary": "Sales Copilot: Get related records",
+        "description": "Sales Copilot: Get related records.",
+        "operationId": "scp-get-related-records",
+        "x-ms-no-generic-test": true,
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
```

Comment:
Could you please fix swagger validation errors? please refer the swagger schema file, this would help to fix the issue.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/apiDefinition.swagger.schema.json

Reply:
I fixed the errors found in the log file. Can you please validate if all the swagger errors are fixed?